packages/@okta/i18n/src/properties/login.properties

```diff
@@ -950,13 +950,20 @@ oie.try.again = Try again
 ## Device Enrollment Terminal
 enroll.title.oda = Download Okta Verify
 enroll.title.mdm = Additional setup required
-enroll.explanation.p1 = To sign in using Okta Verify, you will need to set up Okta Verify on this device. Download the Okta Verify app on {0}.
+enroll.explanation.p1 = To sign in using Okta Verify, you will need to set up Okta Verify on this device.
```

Comment:
Lint is going to fail for this. You will need to make sure you run the grunt task to generate the ok_PL properties file as well. 

Also are these customer facing changes for OIE ? Going forward for any EA/GA feature changes that are customer facing we need to make sure merge the translation PR along with the code changes to make sure i18n is supported.

https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2197816886/SIW+Translation+Request